plugins/GSdx/GSDevice.cpp

```diff
@@ -118,6 +144,19 @@ void GSDevice::Present(const GSVector4i& r, int shader)
 			ShaderConvert_COMPLEX_FILTER}; // FIXME
 
 		Present(m_current, m_backbuffer, GSVector4(r), s_shader[shader]);
+
+static std::chrono::system_clock::time_point timer;
```

Comment:
This is just some test code to exercise the new code and will be removed in the final pull request.  I intentionally didn't indent those two lines to call out the fact that they are static and will not re-initialize between function calls (i.e. global variables).